### ISSUE TITLE
关于log里的损失含义

### ISSUE THREAD

**Issue by yzd-v** [jingyang2017/KD_SRRL#6]
我看到你Log里的损失名是loss, loss_t1, loss_t2, statlosses。而代码里写的是loss,ce,kd。想问一下他们的对应关系

**Comment by jingyang2017** [comment]
log里面的loss是代码里面的ce.      loss_t1是teacher network的cross entropy loss（F.cross_entropy(pred_t, target)，代码里面没写.     loss_t2是代码里面的 F.mse_loss(pred_sc, pred_t). statloss是代码里面的loss_stat

**Comment by jingyang2017** [comment]
> 我看到你Log里的损失名是loss, loss_t1, loss_t2, statlosses。而代码里写的是loss,ce,kd。想问一下他们的对应关系

log里面的loss是代码里面的ce. loss_t1是teacher network的cross entropy loss（F.cross_entropy(pred_t, target)，代码里面没写). loss_t2是代码里面的 F.mse_loss(pred_sc, pred_t). statloss是代码里面的loss_stat
